frontend/src/app/features/in-app-notifications/center/in-app-notification-center.component.html

```diff
@@ -1,11 +1,6 @@
 <div class="op-ian-center">
-  <div class="op-ian-center--loading-indicator" *ngIf="loading$ | async">
-    <op-content-loader>
-      <svg:rect x="0" y="0" width="100%" height="100%" rx="1" />
-    </op-content-loader>
-  </div>
   <div class="op-ian-center--content">
-    <ng-container *ngIf="(hasNotifications$ | async); else noResults">
+    <ng-container *ngIf="dataLoaded || (hasNotifications$ | async); else noResults">
```

Comment:
Since `dataLoaded` is initially set to `true` this will immediately fire although the data cannot be loaded yet. The problem is that `hasNotifications$` fires `false` initially, although there are notifications. So instead you might need to wait for `loading$` and then use the `notifications$` observable directly (which is needed a few lines below anyway). So something like this:
```
<ng-container *ngIf="(loading$ | async) === false">
<ng-container *ngIf="(notifications$ | async) as notifications; else noResults">
        ...
        <op-in-app-notification-entry
            *cdkVirtualFor="let records of notifications; trackBy: ..."
        ...
```

Reply:
Hi @HDinger I think there is a problem here, when  we don't have any notifications, it doesn't show noResult box.